daemon/exec_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
+
+package daemon
+
+// checkExecSupport returns an error if the exec driver does not support exec
+func checkExecSupport(DriverName string) error {
+	return nil
```

Comment:
Exec is supported by the Windows driver, hence returning nil is the right thing here.